app/controllers/application_controller.rb

```diff
@@ -56,8 +55,11 @@ def after_login_redirect(url_to_redirect = nil)
     redirect_to(url_to_redirect) && return unless url_to_redirect.nil?
     if current_website
       redirect_to errors_url, notice: "Logged in"
+    elsif !current_user.websites.empty?
+      set_website(current_user.websites.first)
+      redirect_to errors_url
```

Comment:
Well in that situation we should set the website in the auth_controller when you go in from the github button not here. I can see how this could be convenient but i don't wanna get into weird issues because of this. We will be calling this method everywhere so it should just redirect and nothing else.